src/helper.ts

```diff
@@ -107,9 +95,9 @@ export function generateConfigs(configs: ConfigsOptions = {}): string {
   }
 
   return stringify({
-    sdk_platform: 'js',
-    sdk_version: __VERSION__,
-    ...snakeCaseConfigs
```

Comment:
Previously when `snakeCaseConfigs` contains `sdk_*: undefined`, it overwrites the [Line 110](https://github.com/moneytree/mt-link-javascript-sdk/pull/124/files#diff-a09b7f67f5f992f7ef9d9299f6fac774fb218572c9c2960eb2c30293067a8843L110) and [111](https://github.com/moneytree/mt-link-javascript-sdk/pull/124/files#diff-a09b7f67f5f992f7ef9d9299f6fac774fb218572c9c2960eb2c30293067a8843L111), hence no SDK information was included.